source/agora/common/TransactionPool.d

```diff
@@ -70,7 +69,6 @@ public class TransactionPool
         if (db_exists)
             logInfo("Loading database from: %s", db_path);
 
-        // note: can fail. we may want to just recover txes from the network instead.
         this.db = Database(db_path);
```

Comment:
Please replace this note with a github issue, since the error isn't handled yet.

Reply:
Done: https://github.com/bpfkorea/agora/issues/196